src/providers/completionProvider.ts

```diff
@@ -51,13 +52,16 @@ const priorityMap = {
   choice: 2,
 };
 
-let DUMMY_MAPPING_CHARACTERS;
+let DUMMY_MAPPING_CHARACTERS: string;
+let IS_PLAYBOOK: boolean;
```

Comment:
Any particular reason we're using this naming convention?

Reply:
The initial assumption was that these vars would be constants, but based on the logic, these vars have to be changed. I will update the names. Thanks!

Reply:
@priyamsahoo would you be updating the variable name?